.github/workflows/release.yaml

```diff
@@ -143,10 +142,9 @@ jobs:
           PLUGIN_QUAY_REPO="${{ github.event.inputs.plugin_quay_repository }}"
         fi
 
-        PLUGIN_QUAY_TAG="$PLUGIN_QUAY_REPO:$RELEASE_VERSION"
+        PLUGIN_QUAY_TAG="$PLUGIN_QUAY_REPO:$RELEASE_VERSION $PLUGIN_QUAY_REPO:$BRANCH_VERSION"
```

Comment:
Just some clarification on this line: It is right to have a tag with blanks?

Reply:
Yes, the blank means that you will have two tags (the release version and the branch version, for example v1.78.0 and v1.78). It is exactly what we have in kiali release process (https://github.com/kiali/kiali/blob/24ca2f7ade3222a99664654715286e420a4325a6/.github/workflows/release.yml#L146)